init.ps1

```diff
@@ -4,20 +4,12 @@ Initializes your Rivet working directory for development.
 #>
 [CmdletBinding()]
 param(
-    [string]
+    [Parameter(Mandatory)]
     # The name of the SQL Server tests should run against. Ignored if running under AppVeyor.

```

Comment:
Remove the second sentence since this script has no control over that behavior anymore?